### PR TITLE
Fix issue #489

### PR DESCRIPTION
**Pull request by mateuszbaran**

### CODE REVIEW

File: test/groups/circle_group.jl [hunk 1]
```diff
@@ -1,6 +1,10 @@
 include("../utils.jl")
 include("group_utils.jl")
 
+# TODO: remove after bug in StaticArray is fixed
+@inline Base.copy(a::SizedArray) = __copy(a)
+@inline __copy(a::SizedArray{S,T}) where {S,T} = SizedArray{S,T}(copy(a.data))
```
Comment (kellertuer) [inline test/groups/circle_group.jl:6]: just a short question - why the two `_`?

**Review by kellertuer — APPROVED** [review]

File: test/groups/circle_group.jl (hunk 1 above)
Comment (mateuszbaran) [reply test/groups/circle_group.jl:6]: No real reason, I just felt like it's less likely to extend a function we already have somewhere else (but most likely don't).

Comment (kellertuer) [reply test/groups/circle_group.jl:6]: Ok, well we could also just use `_copy` - just for clarity (others might also wonder why we use `__` here; but I leave this up to you :)

Comment (mateuszbaran) [reply test/groups/circle_group.jl:6]: I'm going to delete it when https://github.com/JuliaArrays/StaticArrays.jl/pull/1037 is merged anyway so it's just a temporary fix.

**Pull request merged**